packages/dashboard/e2e/auth-ready.js

```diff
@@ -15,13 +15,70 @@ async function authReady(timeout = 30000) {
     }, timeout);
     let retryTimer;
     const waitAuthReady = () => {
-      req = http.request('http://localhost:8080/auth/', () => {
+      // initialize network and container related variables.
+      let authIpAddress;
+      const commonNetwork = 'auth_dashboard_network';
+      const commonNetworkSubnet = '172.16.0.0/16';
+      const authContainerIp = '172.16.0.2';
+
+      // check if we are in github CI environment
+      if (process.env.CI) {
+        // check if auth container has already been initialized
+        let authContainer = execSync(
+          'docker ps -q --filter ancestor=romi-dashboard/auth',
+        ).toString();
+        if (authContainer) {
+          process.env.AUTH_CONTAINER = authContainer;
+
+          // check if auth container is already connected to custom network to prevent running the same operation if it has
```

Comment:
nit: if it has...

Maybe if it has is a leftover or I think something is missing here.